docker/docker-jaxrs-dockerfile/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:jre-alpine
```

Comment:
Perhaps use the same base image as the `docker/docker-jaxrs` example? Not sure which one is "more official" :-) It probably doesn't really matter.

Reply:
Note that the Dockerfile existed before, I just had to move it, but didn't alter the base image.

Reply:
Ah, didn't notice that, as Git[Hub] didn't recognize it's a file move :-)

Reply:
Yes, I think that's because I had to add one argument to the entry point (it failed without that).